lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -866,6 +865,30 @@ def test_check_coordinate_variable_no_fill_value(self):
         self.assertEqual(len(ret_val), 1)
         self.assertFalse(ret_val[0].value)
 
+    def test_check_time_variable(self):
+        time_att = [('TIME',),
+                    ('TIME', 'standard_name'),
+                    ('TIME', 'axis'),
+                    ('TIME', 'calendar'),
+                    ('TIME', 'valid_min'),
+                    ('TIME', 'valid_max'),
+                    ('TIME', 'units')]
+
+        ret_val = self.imos.check_time_variable(self.new_dataset)
+        self.assertEqual(len(ret_val), len(time_att))
+        passed_att = [r.name[1:] for r in ret_val if r.value]
+        self.assertEqual(len(passed_att), len(time_att))
+        self.assertEqual(set(passed_att), set(time_att))
```

Comment:
`assertEqual(len(...), len(...))` makes sure we have the right number of passed attributes.
`assertEqual(set(...), set(...))` makes sure we all the correct passed attributes, but doesn't catch a case where there are multiple return results with the same attribute name (since a `set` has only unique values). In this case it probably is a bit redundant, but I don't think it hurts.